chore: cow explorer improvements (#245)
cowprotocol/docs: docs/cow-protocol/tutorials/cow-explorer/order.md

</details>

## Partially fillable

A more complex example may be a user who has placed a _partially fillable_ order. This means that the user has expressed a swap intent for a certain amount of tokens, but the order may be filled in multiple batches. This can only be used with [Limit orders](../cow-swap/limit).

![Partially fillable order overview](/img/explorer/order_partial_overview.png)

> By clicking the 'View fills' button, the user can see the fills that have been executed for this order. Fills for the above example are show below:

Reviewer: Typo?

```suggestion
By clicking the 'View fills' button, the user can see the fills that have been executed for this order. Fills for the above example are shown below:
```

![Partially fillable order fills](/img/explorer/order_partial_fills.png)

Here we can see that the order has been 100% filled, but in dozens of batches. For ease of analysis, an overview is shown providing:

- The total amount filled (in this case 100%)
- The average execution price (in this case 0.22134041 ANT for DAI)
- Total surplus (in this case 0.39% / 860.894 DAI)
- The limit price applied across all fills (in this case 0.22222222 ANT for DAI)

## States

As CoW Explorer is meant to help users visualize their signed orders, a critical parameter to show is the _state_ of the signed order. This parameter has the following status:

> - **Open** (pending)**:** State for a standing order. Orders in this state are being considered by the solvers. It is the entry state for all orders, the default from the moment the order is placed. From here the order can transition to all other states.

Reviewer: Would be worth to add a note about pre-signing orders?
For those types of orders, `Open` is not the entry state.

(Note, I don't know how to create foot notes, this is just a suggestion on the content)

```suggestion
- **Open** (pending)**:** State for a standing order. Orders in this state are being considered by the solvers. It is the entry state for all orders*, the default from the moment the order is placed. From here the order can transition to all other states.

^* Except for pre-sign orders, where the first state is [Pre-signing](link to pre-sign entry)
```

Author: Footnote added. Unfortunately can't link easily back up, but given that it's so close, should be OK.

- **Filled:** State for an executed/settled order. Orders in this state have been executed and the corresponding funds transferred to the target account. Orders in this state will have available a link to the corresponding Etherscan settlement transaction. This settlement transaction would contain your order's execution and any other orders that are part of the same batch.
- **Expired:** State for orders that have not been executed. Orders in this state have not been executed during the defined expiration time.
- **Cancelled:** State for orders that have been cancelled. Orders in this state have been cancelled as per user request.
- **Partially-filled:** State for orders that have been partially executed. Parts of the order have been executed and the corresponding funds transferred to the target account.

> - **Pre-signing:** State for orders that are to be pre-signed. Commonly used for orders submitted through the [CoW Swap](../cow-swap) `Safe` interface.

Reviewer: I would mention it's the case for any SC wallet

```suggestion
- **Pre-signing:** State for orders that are to be pre-signed. Commonly used for orders submitted from a Smart Contract wallet, such as the [Safe](https://safe.global/) wallet.
```
